test/grouping.jl

```diff
@@ -2,14 +2,97 @@ module TestGrouping
     using Base.Test
     using DataTables
 
+    srand(1)
     dt = DataTable(a = repeat([1, 2, 3, 4], outer=[2]),
                    b = repeat([2, 1], outer=[4]),
                    c = randn(8))
     #dt[6, :a] = Nullable()
     #dt[7, :b] = Nullable()
 
-    cols = [:a, :b]
+    nullfree = DataTable(Any[collect(1:10)], [:x1])
+    # colwise(::Vector{<:Function}, ::AbstractDataTable)
+    fxn = [sum]
```

Comment:
> it serves both as a header for the test group

That signifies to me that these should be `@testset`s instead. That way the grouping is entirely unambiguous and the results are well structured.

> it can be reused

I find reusing variables a somewhat dangerous practice in tests. That's another thing that's solved with `@testset`s, since each introduces a new scope.